packages/workspace/src/browser/workspace-input-dialog.ts

```diff
@@ -32,8 +33,10 @@ export class WorkspaceInputDialog extends SingleTextInputDialog {
     constructor(
         @inject(WorkspaceInputDialogProps) protected readonly props: WorkspaceInputDialogProps,
         @inject(LabelProvider) protected readonly labelProvider: LabelProvider,
+        @inject(WorkspaceService) protected readonly workspaceService: WorkspaceService
```

Comment:
I think the following should be optional as not to break any current callers or extenders.